### PR TITLE
View list of collection instruments uploaded

### PR DESCRIPTION
**Pull request by benjefferies**
* User story *
As a Collection Exercise Coordinator
I need: to view the collection instruments that the system holds for the
given collection exercise
So that: I am happy that the collection instrument(s) loaded are correct

* Acceptance criteria *
Given the collection instruments have been loaded
When the user navigates to the collection exercise details page
Then they are able to see the filename (surveyid_formtype_period) of all
collection instruments that have been loaded

### CODE REVIEW

File: response_operations_ui/controllers/collection_instrument_controllers.py [hunk 1]
```diff
@@ -15,3 +16,12 @@ def upload_collection_instrument(short_name, period, file):
...
         raise ApiError(response)
+
+
+def get_collection_instruments(short_name, period):
+    logger.debug('Getting collection instruments', short_name=short_name, period=period)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/collection-instrument/{short_name}/{period}'
+    response = requests.get(url)
+    if response.status_code != 200:
+        raise ApiError(response)
+    return json.loads(response.text)
```
Comment (andrewmil) [inline response_operations_ui/controllers/collection_instrument_controllers.py]: A debug level log line saying getting the collection instruments was successful here would be good for consistency with the other controller functions

File: response_operations_ui/views/collection_exercise.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+from response_operations_ui.controllers import collection_exercise_controllers
+from response_operations_ui.controllers import collection_instrument_controllers
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@app.route('/surveys/<short_name>/<period>', methods=['GET'])
+def view_collection_exercise(short_name, period):
+    ce_details = collection_exercise_controllers.get_collection_exercise(short_name, period)
+    cis = collection_instrument_controllers.get_collection_instruments(short_name, period)
```
Comment (andrewmil) [inline response_operations_ui/views/collection_exercise.py]: I think there's a question around whether getting `cis` should be a separate call or if it should be returned as part of `ce_details` from the original backstage call. 
The pattern we've been using up til now would be to return it in the same call

File: tests/test_data/collection_exercise/collection_exercise_details.json [hunk 3]
```diff
@@ -30,5 +30,19 @@
...
+          "e33daf0e-6a27-40cd-98dc-c6231f50e84a"
+        ],
+        "RU_REF": [],
+        "SURVEY_ID": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
+      },
+      "file_name": "collection_instrument.xlsx",
+      "id": "f732afbe-c710-4c95-a8d3-6644833195a7",
+      "surveyId": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
+    }
+  ]
```
Comment (rhiGriff) [inline tests/test_data/collection_exercise/collection_exercise_details.json]: No new line at end of file.

File: tests/test_data/collection_exercise/collection_exercise_details_no_ci.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "survey": {
+    "id": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87",
+    "longName": "Business Register and Employment Survey",
+    "shortName": "BRES",
+    "surveyRef": "221"
+  },
+  "collection_instruments": [
+  ]
+}
```
Comment (rhiGriff) [inline tests/test_data/collection_exercise/collection_exercise_details_no_ci.json]: No new line at end of file.

File: tests/test_data/collection_exercise/collection_exercise_details.json (hunk 3 above)
Comment (benjefferies) [reply tests/test_data/collection_exercise/collection_exercise_details.json]: Don't think it's necessary in this case as it's json and it's only pep8 thing

File: tests/test_data/collection_exercise/collection_exercise_details_no_ci.json (hunk 4 above)
Comment (benjefferies) [reply tests/test_data/collection_exercise/collection_exercise_details_no_ci.json]: Don't think it's necessary in this case as it's json and it's only pep8 thing

**Pull request merged**